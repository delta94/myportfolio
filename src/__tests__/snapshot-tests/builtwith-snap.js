import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import BuiltWith from '../../components/builtwith';

it('aboutme renders correctly', () => {
  const wrapper = shallow(<BuiltWith tech={['react', 'typescript', 'gatsby']} />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
