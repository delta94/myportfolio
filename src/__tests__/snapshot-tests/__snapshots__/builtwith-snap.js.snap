// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aboutme renders correctly 1`] = `
<styled.div>
  <span
    style={
      Object {
        "fontSize": "0.8rem",
        "fontWeight": "bold",
      }
    }
  />
  <styled.div
    title="Built with react, typescript, gatsby"
  >
    <styled.div
      key="react"
      name="react"
    />
    <styled.div
      key="typescript"
      name="typescript"
    />
    <styled.div
      key="gatsby"
      name="gatsby"
    />
  </styled.div>
</styled.div>
`;
